pkg/docker/docker.js

```diff
@@ -108,6 +108,28 @@ define([
         return promise;
     };
 
+    docker.check_image = function check_image(image) {
+        var dfd = $.Deferred();
+            /*
+             * Gets image info locally
+             */
+            http.get("/v1.12/images/" + image + "/json").
```

Comment:
The indentation here is broken. In addition, I think we need to encodeURIComponent() the image in this URL.